pandas/core/arrays/interval.py

```diff
@@ -749,7 +759,9 @@ def shift(self, periods: int = 1, fill_value: object = None) -> "IntervalArray":
 
         empty_len = min(abs(periods), len(self))
         if isna(fill_value):
-            fill_value = self.left._na_value
+            from pandas import Index
+
+            fill_value = Index(self._left, copy=False)._na_value
```

Comment:
why do you need to wrap? doesn't this *always* have the same _na_value as the Array?

Reply:
self._left can be an ndarray, which doesnt have _na_value